canonical-web-and-design/hiring.canonical.com
# Update pending chart's link's url

Text of the link should be fixed.

## Done
Pending chart's link is updated. 

## QA

- Run server: `dotrun`
- View results at `http://0.0.0.0:8501/`
- Click to "Applications Review"
- Verify the application count is displayed correctly.
- Apply a filter.
- Verify the application count is displayed correctly. (It should change)
- Click on the link
- Verify Greenhouse's page is opened and candidate list is displayed.
- Go to "Written Submissions" page
- Verify two counts are displayed. (One for submitted one for not submitted written submissions' count)
- Click on the link
- Verify Greenhouse's page is opened and candidate list is displayed.

## Issue
Fixes workplace-engineering-squad #146
